src/components/dataTable/utils.js

```diff
@@ -27,7 +27,7 @@ export const getIdentifier = (subject) => {
 
 export const getSingleReferenceUrl = (pubModId) => {
   let url;
-  if(pubModId.includes("PMID")){
+  if(pubModId.includes("PMID") || pubModId.includes("ORPHA")|| pubModId.includes("ORPHA")){
```

Comment:
Does this need to be checked for twice?

Reply:
@oblodgett - I believe the appropriate check is now in place